netsim/modules/srv6.py

```diff
@@ -1,29 +1,101 @@
 #
 # SRv6 transformation module
 #
+import typing
+
 from box import Box
+from ..augment import addressing,devices
 
 from . import _Module
 from ..utils import log
+from .. import data
 import ipaddress
 
-class SRV6(_Module):
+DEFAULT_VPN_AF: typing.Final[dict] = {
+  'ipv4': [ 'ibgp' ],
+  'ipv6': [ 'ibgp' ]
+}
 
-  def node_post_transform(self, node: Box, topology: Box) -> None:
+POOL_NAME = "srv6_locator"
```

Comment:
You could use `data.global_vars.get_const` instead of a hardcoded value (but probably within a function, I'm not sure the topology is already set up yet when the module is loaded)

Reply:
added